source/Ship.cpp

```diff
@@ -3203,6 +3205,20 @@ double Ship::HullUntilDisabled() const
 
 
 
+const ShipJumpNavigation &Ship::JumpNavigation() const
+{
+	return navigation;
+}
+
+
+
+void Ship::RecalibrateJumpNavigation()
```

Comment:
How is this function neccessary?
If you do this some other short cuts would also be helpful.

Reply:
Since `Recalibrate` needs `*this`, it's easier to handle recalibration this way as opposed to needing to potentially do something like `ship->JumpNavigation().Recalibrate(ship.get())` in outside classes. That is, with this method, Ship passes itself to JumpNav in the same way every time, whereas outside of this we would need to pass Ship to JumpNav differently depending on how the Ship is being stored (is it a reference, pointer, or shared pointer?). None of the other functions have that same issue, hence why I didn't make short cuts for anything else.